frontend/src/pages/IstioConfigList/IstioConfigListPage.tsx

```diff
@@ -139,18 +140,44 @@ class IstioConfigListPageComponent extends FilterComponent.Component<
 
   // Fetch the Istio configs, apply filters and map them into flattened list items
   fetchIstioConfigs(namespaces: string[], typeFilters: string[], istioNameFilters: string[]) {
-    return this.promises
-      .registerAll(
-        'configs',
-        namespaces.map(ns => API.getIstioConfig(ns, typeFilters, true, '', ''))
-      )
-      .then(responses => {
-        let istioItems: IstioConfigItem[] = [];
-        responses.forEach(response => {
-          istioItems = istioItems.concat(toIstioItems(filterByName(response.data, istioNameFilters)));
-        });
-        return istioItems;
-      });
+    // return this.promises
+    //   .registerAll(
+    //     'configs',
+    //     // THIS should be one call, not per namespace , OR the next section
+    //     namespaces.map(_ => API.getAllIstioConfigs(namespaces, typeFilters, true, '', ''))
+    //   )
+    //   .then(responses => {
+    //     let istioItems: IstioConfigItem[] = [];
+    //     responses.forEach(response => {
+    //       namespaces.forEach(ns => {
+    //         istioItems = istioItems.concat(toIstioItems(filterByName(response.data[ns], istioNameFilters)));
+    //       })
+    //     });
+    //     return istioItems;
+    //     });
+    // OR PER Chunk
+    let istioItems: IstioConfigItem[] = [];
+    _.chunk(namespaces, 10).forEach(chunk => {
```

Comment:
I've started to review this design, but not sure if I understand it well.
I guess you want to apply in the app/services/workload/istio-config list pages the design to fetch in "chunks" 10 by 10.

It's not a bad idea, but I'd say it's a different use case than the overview.

On the overview page, by default, it fetches all namespaces, and only when it filters, the search space is narrowed.

But in the app/services/workloads we moved to have the health config in a single namespace but still filtered by namespace

![image](https://user-images.githubusercontent.com/1662329/175537437-4d46534a-3e90-4cc1-a91f-802f7493cc54.png)

Perhaps, I'd leave the app/service/workload/istio-config list pages for a next iteration, not this work, and I would concentrate in the overview page.

In the next iteration, we can study if we refactor those endpoints to have all of them a list of namespaces passed as parameters (like the graph) instead of a URL, but I see it weird to start doing this on the Istio Config list page and not others.